### ISSUE TITLE
Use Gitflow Workflow for development

### ISSUE THREAD

**Issue by cminhho** [webteamr-d/react-redux-saga-exercise#1]
A **develop** branch is created from master
A **release** branch is created from develop
**Feature branches** are created from develop
When a feature is complete it is merged into the develop branch
When the release branch is done it is merged into develop and master
If an issue in master is detected a **hotfix** branch is created from master
Once the hotfix is complete it is merged to both develop and master
